xml/deployment_installing_nodes.xml

```diff
@@ -993,15 +993,15 @@
      the cluster.
     </para>
     <important>
-     <title>Leave the admin node configuration open</title>
+     <title>Do Not Close The Velum Configuration</title>
      <para>
       If you quit out of the configuration process at this time by closing
       the browser window/tab, you will lose all input you made to the settings
       and have to restart the configuration from the beginning.
      </para>
      <para>
-      We recommend you leave the browser window/tab open while you perform the
-      installation of the other nodes in a new window.
+      You must leave the browser window/tab open while performing the
+      installation of the other nodes in a new window/tab.
```

Comment:
I don't think the new or the old descriptions are accurate. On every step of the setup (when you click `Next`), we save that information in the database, and it's perfectly safe to close it and come to it afterwards. You will only lose whatever was on that page where you didn't click `Next`, but the previous steps data will survive across sessions.

Reply:
Yes, they should be saved, but only when you cilck on `Next`.